docs/source/basics.rst

```diff
@@ -176,6 +176,25 @@ There is more information about creating stubs in the
 The following sections explain the kinds of type annotations you can use
 in your programs and stub files.
 
+Mypy will also look in some default install directory to try to find annotations.
+For example, for UNIX based, with python 3.4, it will try to find it in
+- ``${HOME}/.local/shared/typehints/python3.4/``
+- ``/usr/shared/typehints/python3.4/``
```

Comment:
The standard directory is `/usr/share/` (no `d`). And I think it would be better to put `python*` before `typehints` in the path, to mirror `/usr/lib/pythonX.Y/`. I'd also rather use `/usr/lib/python*` for this rather than introduce a second python tree under `/usr/share` (and maybe put it under `site-packages`?).